scripts/steam.js

```diff
@@ -200,7 +201,11 @@ module.exports = robot => {
         } else {
           price = data.price === 0 ? 'Free-To-Play' : data.final
         }
-        fields.splice(1, 0, `Valor: *${numberToCLPFormater(price, 'CLP $')}*${discount}`)
+        if (price === 'Free-To-Play') {
```

Comment:
Estás seguro que este es el unico valor no numerico de `price` ? Si no lo haria al contrario:

```
if (!isNaN(parseFloat(price)) && isFinite(price) {
  ...
}
```

Reply:
Si, seteo el 'Free-To-Play' después de recibir el valor numérico. 

Queda mucho mejor de esa forma, lo cambiaré :)